cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/MockETLHandler.java

```diff
@@ -63,8 +63,23 @@ public void getTemplates(HttpRequest request, HttpResponder responder) throws Ex
   public void getTemplate(HttpRequest request, HttpResponder responder, @PathParam("template-id") String templateId)
     throws Exception {
     JsonObject template = new JsonObject();
+    JsonArray properties = new JsonArray();
     template.addProperty("name", templateId);
     template.addProperty("description", "Template used to create ETL Adapters");
+    if (templateId.equalsIgnoreCase(BATCH)) {
+      JsonObject scheduleProperty = new JsonObject();
```

Comment:
We should add a type as well? Let's look at what Coopr returns and return relevant info for UI to use.